app/src/main/res/layout/dialog_cancel_relation.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:orientation="vertical"
+              android:background="#f9f6e5"
+              android:layout_height="match_parent">
+
+    <TextView android:layout_width="match_parent"
+              android:layout_marginTop="8dp"
+              android:textSize="24sp"
+              android:layout_marginLeft="16dp"
+              android:textColor="#7e1b59"
```

Comment:
I think you should define this color in **colors.xml** file and give a reference to that color here instead of hardcoding the value of color here only.